### PR TITLE
addpatch: web-ext

### PR DESCRIPTION
**Pull request by kxxt**
- Disable V8's buggy JIT that makes eslint and babel fail <https://github.com/babel/babel/issues/15501>
  - Upstream bug report: https://bugs.chromium.org/p/v8/issues/detail?id=13836
- Increase test timeout

### CODE REVIEW

File: web-ext/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,34 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -18,13 +18,20 @@ conflicts=('nodejs-web-ext')
+ options=('!strip')
+ # tarball on npmjs lacks scripts for building from sources
+ source=("https://github.com/mozilla/web-ext/archive/$pkgver/web-ext-$pkgver.tar.gz"
+-        "nodejs19.diff")
++        "nodejs19.diff"
++	"web-ext-no-jit.patch")
```
Comment (Xeonacid) [inline web-ext/riscv64.patch:9]:
```suggestion
+        "nodejs19.diff"
+        "web-ext-no-jit.patch")
```

**Review by Xeonacid — CHANGES_REQUESTED** [review]

**Review by Xeonacid — APPROVED** [review]

**Comment by yan12125** [comment]
> Increase test timeout

FWIW, I incorporated this change into official PKGBUILD https://gitlab.archlinux.org/archlinux/packaging/packages/web-ext/-/blob/ce832fe576d14ae2e46bc8e20c7c222f42f89042/PKGBUILD#L48

> Disable V8's buggy JIT that makes eslint and babel fail https://github.com/babel/babel/issues/15501

This change is dropped in https://github.com/felixonmars/archriscv-packages/pull/2465

**Comment by kxxt** [comment]
> FWIW, I incorporated this change into official PKGBUILD https://gitlab.archlinux.org/archlinux/packaging/packages/web-ext/-/blob/ce832fe576d14ae2e46bc8e20c7c222f42f89042/PKGBUILD#L48

Thanks very much!

**Comment by yan12125** [comment]
web-ext 7.9.0-1 is now in [extra] and RISC-V patches failed as it's already applied: https://archriscv.felixc.at/.status/log.htm?url=logs/web-ext/web-ext-7.9.0-1.log

**Comment by Xunop** [comment]
> web-ext 7.9.0-1 is now in [extra] and RISC-V patches failed as it's already applied: https://archriscv.felixc.at/.status/log.htm?url=logs/web-ext/web-ext-7.9.0-1.log

I have removed this patch: https://github.com/felixonmars/archriscv-packages/pull/3414

Thank you for your attention.

**Comment by yan12125** [comment]
Thanks!

**Pull request merged**